### DISCUSSION TITLE
Only show specified Validation Fields

### DISCUSSION THREAD

**Discussion by cbaxter7** [vueform/builder#75] [Questions] [answered]
I'm looking to strip down the validation fields to only display the Required toggle, is there a straight forward way to do this?

At the moment I'm currently using the ValidationField which displays all of the fields below
![image](https://github.com/user-attachments/assets/5511dc06-5bf6-4add-964d-7d4a369b5c67)

**Comment by adamberecz** [comment, marked as answer]
There is! Check out this part of the docs, especially around the 'validation' keyword:
https://builder.vueform.com/docs/config-panels#element-settings-panel

Also, here you can find all the available validators globally:
https://builder.vueform.com/docs/config-panels#default
